### ISSUE TITLE
Popup theme, wrong tabid and no antiForgeryToken

### ISSUE THREAD

**Issue by ohine** [dnnsoftware/Dnn.AdminExperience#4]
I am developing an angular app, in MVC, using web api, and this works fine in a NON popup theme, such as Admin and Home (Xcillion). In a popup theme the tabid is set to -1 and there is no element RequestVerificationToken and thus using sf to get the antiForgeryToken OR window.document.getElementsByName('__RequestVerificationToken')[0].value is not possible; this occurs only for non administrators. When logged in as super user this issue is not present.

In summary:
1. MVC, DNN popup theme and Non administrator: the above mentioned problem occurs. Using dnn9 with the default theme (Xcillion).
2. In other scenarios the tabid is set correctly and the antiForgeryToken is present.

Originally created by @kurtwilbies at https://github.com/dnnsoftware/Dnn.AdminExperience.Library/issues/9

**Comment by valadas** [comment]
Moved this to https://github.com/dnnsoftware/Dnn.Platform/issues/2548
